KOService.Domain/Entities/Activity.cs

```diff
@@ -46,5 +50,14 @@ protected override void Open()
             base.Open();
             SetStatus(ActivityStatus.Open);
         }
+        public void AssignMechanic(Guid mechanicId)
+        {
+            if (MechanicId == null)
+            {
+                MechanicId = mechanicId;
+                Open();
```

Comment:
W employeeTask w konstruktorze, już wywołane jest Open(). Rozumiem, że to jest błąd, ponieważ status powinien byc "Open" dopiero gdy przypiszemy mechanika. W takim razie trzeba to usunąć z konstruktora EmployeeTask i dodać do konstruktora Repair. Jakbyś mógł to zrobić teraz to by było spoko :P